cromshell

```diff
@@ -711,21 +711,23 @@ function slim-metadata()
 # Get the status of the given job and how many times it was run
 function execution-status-count() 
 {
-  assertCanCommunicateWithServer $2
+  assertCanCommunicateWithServer "$2"
   turtle
   f=$( makeTemp )
-  curl --connect-timeout $CURL_CONNECT_TIMEOUT --max-time $CURL_MAX_TIMEOUT --compressed -s ${2}/api/workflows/v1/$1/metadata?${CROMWELL_METADATA_PARAMETERS} > $f
-  [[ $? -ne 0 ]] && error "Could not connect to Cromwell server." && return 9
+  if ! curl --connect-timeout "$CURL_CONNECT_TIMEOUT" --max-time "$CURL_MAX_TIMEOUT" --compressed -s "$2/api/workflows/v1/$1/metadata?includeKey=executionStatus&includeKey=backendStatus&expandSubWorkflows=true" > "$f" ; then
+    error "Could not connect to Cromwell server." && return 9
+  fi
 
   # Make sure the query succeeded:
-  if [[ "$( cat $f | jq '.status' | sed -e 's#^"##g' -e 's#"$##g' )" == 'fail' ]] ; then
-    reason=$( cat $f | jq '.message' | sed -e 's#^"##g' -e 's#"$##g' )
+  if [[ "$( < "$f" jq '.status' | sed -e 's#^"##g' -e 's#"$##g' )" == 'fail' ]] ; then
```

Comment:
What is the rationale for changing the call to `cat` to the output re-direction?  Is it for speed?